pkgs/by-name/dm/dmarc-report-converter/package.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub, lib, pkgs }:
+
+buildGoModule rec {
+  pname = "dmarc-report-converter";
+  version = "0.6.5";
+
+  src = fetchFromGitHub {
+    owner = "tierpod";
+    repo = pname;
```

```suggestion
    repo = "dmarc-report-converter";
```
please hardcode this, it has nicer semantics when overriding the pname

---

Done.